src/main/java/uk/gov/hmcts/darts/arm/config/ArmDataManagementConfiguration.java

```diff
@@ -3,9 +3,11 @@
 import lombok.Getter;
 import lombok.Setter;
 import org.springframework.boot.context.properties.ConfigurationProperties;
+import org.springframework.boot.context.properties.EnableConfigurationProperties;
 import org.springframework.context.annotation.Configuration;
 
 @Configuration
+@EnableConfigurationProperties(ArmApiConfigurationProperties.class)
```

Comment:
from my knowledge this is needed only if you didn't have `@Configuration` already

Reply:
I've left this as removing it here in our version of spring boot will currently fail all the integration tests e.g.

`Caused by: org.springframework.beans.factory.UnsatisfiedDependencyException: Error creating bean with name 'armApiServiceImpl' defined in file [ArmApiServiceImpl.class]: Unsatisfied dependency expressed through constructor parameter 0: No qualifying bean of type 'uk.gov.hmcts.darts.arm.config.ArmApiConfigurationProperties' available: expected at least 1 bean which qualifies as autowire candidate.`